playground/ssr-vue/vite.config.js

```diff
@@ -40,8 +44,78 @@ export default defineConfig({
           return `export const msg = "[success] from conventional virtual file"`
         }
       }
-    }
+    },
+    {
+      name: 'virtual-module',
+      resolveId(id) {
+        if (id === virtualFile) {
+          return virtualId
+        } else if (id === nestedVirtualFile) {
+          return nestedVirtualId
+        }
+      },
+      load(id) {
+        if (id === virtualId) {
+          return `export { msg } from "@nested-virtual-file";`
+        } else if (id === nestedVirtualId) {
+          return `export const msg = "[success] from conventional virtual file"`
+        }
+      }
+    },
```

Comment:
@patak-dev this plugin it seems is already defined above 👀

Reply:
Ha, good catch! Would you like to do a PR to remove the second one from the config?

Reply:
will do